modules/ROOT/pages/_partials/app-security/using-the-admin-ui.adoc

```diff
@@ -0,0 +1,58 @@
+= Accessing the dashboard
+
+The {app-security-service} Admin console is viewed by opening the console URL in a browser. This URL is seen in the list of available dashboards after the {app-security-service} Service has been provisioned.
```

Comment:
@davidffrench could you let me know if this text is suitable

Reply:
This looks good to me @damienomurchu . Unless it is called out specifically in other parts of the documentation, I would extend this slightly to say `available services in MDC after`

Reply:
:+1: